### PR TITLE
update(multi-tenancy): deprecate and delete content

### PR DESCRIPTION
**Pull request by NathalieC**
* add warning for deprecation
* delete content
* modify description

### CODE REVIEW

File: modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc [hunk 1]
```diff
@@ -1,297 +1,16 @@
...
-
-===== Activate the tenant with id `101`
-
-[source,bash]
-----
-$ curl -v -b saved_cookies.txt -X PUT 'http://localhost:8080/bonita/API/platform/tenant/101' \
--H "Content-Type: application/json" \
--H 'X-Bonita-API-Token: a94cbf84-6b71-409a-981f-f91b17466929' \
--d '{"state":"ACTIVATED"}'
+Starting with Bonita 2021.1, this feature is *deprecated*, and as such, not offered to new customers nor as a Subscription modification for existing customers. +
```
Comment (tbouffard) [inline modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc:7]: ⚠️ there is no mention to such deprecation in the release notes whereas the 2021.1 version has been released a few months ago.
It should at least mention it afterwards and could have a link to this new content for alternatives.

Comment (philippelaumay) [reply modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc:7]: good catch ! 
Nathalie, please add this mention in the RN of 2021.1

Comment (NathalieC) [reply modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc:7]: 👍

**Review by tbouffard — APPROVED** [review]
LGTM and validated by @philippebonita so merging

**Pull request merged**